src/nvme/mi-mctp.c

```diff
@@ -97,6 +102,7 @@ static struct __mi_mctp_socket_ops ops = {
 	recvmsg,
 	poll,
 	ioctl_tag,
+	bind,
```

Comment:
These socket ops are only here for the test framework; with the two socket types now (the message fd, and the aem fd), we might want to shift the abstraction here a little.

For example, we could do something like:

```diff
 static struct __mi_mctp_socket_ops ops = {
-	socket,
+ 	msg_socket,
+       aem_socket,
 	sendmsg,
 	recvmsg,
 	poll,
 	ioctl_tag,
};
```

where `aem_socket` and `msg_socket` provide the ops for creating and setting-up the appropriate socket fd. For the AEM case, that would handle the bind() as well.

This may mean it's easier to do the socket handling for the aem tests, as we know which type of mock socket to create.